internal/contour/route.go

```diff
@@ -113,9 +113,13 @@ func (v *routeVisitor) Visit() map[string]*v2.RouteConfiguration {
 		switch vh := vh.(type) {
 		case *dag.VirtualHost:
```

Comment:
I think we need to update the SecureVirtualHost case as well, to support Aliases for HTTPS.

Reply:
I should have added a comment on this, TLS won't work until #596 is merged. If we get that merged happy to update, or maybe make a new Issue to give TLS alias support.

Reply:
Ah! Got it.